index-scheduler/Cargo.toml

```diff
@@ -12,7 +12,7 @@ license.workspace = true
 
 [dependencies]
 anyhow = "1.0.64"
-bincode = "1.3.3"
+bincode = "2.0.0-rc.2"
```

Comment:
We will merge this PR once the `rc`s are done and released as a final version. What do you think?
They just published a `rc.3` recently on the 30th of March.